src/functions/template-tags/general.php

```diff
@@ -1135,7 +1135,7 @@ function tribe_get_days_between( $start_date, $end_date, $day_cutoff = '00:00' )
 			$end_date->modify( '-1 day' );
 		}
 
-		return Tribe__Events__Date_Utils::dateDiff( $start_date->format( 'Y-m-d ' . $day_cutoff ), $end_date->format( 'Y-m-d ' . $day_cutoff ) );
+		return Tribe__Events__Date_Utils::date_diff( $start_date->format( 'Y-m-d ' . $day_cutoff ), $end_date->format( 'Y-m-d ' . $day_cutoff ) );
```

Comment:
It _is_ changed here: https://github.com/moderntribe/the-events-calendar/blob/release/119/src/Tribe/Date_Utils.php#L102

The change in `Query.php` is uncommitted on my local environment (I missed pushing it with these changes).